plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/editor.scss

```diff
@@ -0,0 +1,40 @@
+.wc-block-product-gallery-thumbnails {
```

Comment:
Naive question, can we achieve the same result with a single `scss` file and not introduce this one?

Reply:
Probably we could but we should not. This is the pattern we use to separate Editor-specific styles from the frontend ones. Basically what's in `style.scss` will be included in Editor and frontend, while `editor.scss` will be applied in Editor only.